src/redist/targets/BundledManifests.targets

```diff
@@ -66,8 +66,13 @@
     </GenerateGuidFromName>
       
     <ItemGroup>
-      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\**\*.json;%(BundledManifests.RestoredNupkgContentPath)\**\*.targets"
+      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\data\**\*"
+                       Condition="Exists('%(RestoredNupkgContentPath)\data')"
                        DestinationPath="%(WorkloadManifestId)/"/>
+
+      <ManifestContent Include="%(BundledManifests.RestoredNupkgContentPath)\**\*.json;%(BundledManifests.RestoredNupkgContentPath)\**\*.targets"
```

Comment:
Doesn't ```**``` include recursive subdirectories, so data would be included by the second pattern?

Reply:
We only want to include things under the data folder if it exists, so this is enforcing that. Also, once blazor starts producing packages with the data dir I'm hoping to remove the second pattern.